python3-modules.json

```
@@ -0,0 +1,68 @@
+{
+    "name": "python3-modules",
+    "buildsystem": "simple",
+    "build-commands": [],
+    "modules": [
+        {
+            "name": "python3-lxml",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"lxml\" --no-build-isolation"
```

it's subtle, but this will fail: lxml is in the SDK (so the installaton will be ignored or fail) and the platform doesn't have it so it will be missing.

```suggestion
                "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"lxml\" --no-build-isolation --ignore-installed"
```

---

This was originally generated by the [flatpak-pip-generator](https://github.com/flatpak/flatpak-builder-tools/tree/master/pip). So this should probably addressed in there as well then.